fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -269,6 +286,7 @@ class MirabufSceneObject extends SceneObject implements ContextSupplier {
         }
 
         const rigidNodes = this._mirabufInstance.parser.rigidNodes
+        // for pies nodeToBody is empty
```

Comment:
?

Reply:
"pieces" not "pies" lol, I'll remove the comment.